src/main/java/org/littleshoot/proxy/extras/SelfSignedSslEngineSource.java

```diff
@@ -150,8 +234,7 @@ public void checkServerTrusted(X509Certificate[] arg0,
             sslContext = SSLContext.getInstance(PROTOCOL);
             sslContext.init(keyManagers, trustManagers, null);
         } catch (final Exception e) {
-            throw new Error(
-                    "Failed to initialize the server-side SSLContext", e);
+            throw new Error("Failed to initialize the server-side SSLContext", e);
```

Comment:
Actually I changed nothing here, it 's not my decision to throw an Error here. I think it 's fine  and acceptable.